src/Diagnostics/FxCop/System.Runtime.Analyzers/Test/Design/DefineAccessorsForAttributeArgumentsTests.Fixer.cs

```diff
@@ -131,12 +131,12 @@ public InternalGetterTestAttribute(string name)
     public string Name
     {
         get { return m_name; }
-        set { m_name = value; }
+        internal set { m_name = value; }
     }
 }");
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/662"), Trait(Traits.Feature, Traits.Features.Diagnostics)]
+        [Fact, Trait(Traits.Feature, Traits.Features.Diagnostics)]
         public void CSharp_CA1019_MakeGetterPublic2()
         {
             VerifyCSharpFix(@"
```

Comment:
For this test, shouldn't the code fix change the setter's accessibility to internal to preserve its original accessibility?